### PR TITLE
Update modeler documentation

### PR DESCRIPTION
**Pull request by barmac**
[](https://app.camunda.com/jira/browse/)

This PR will stay to be a draft until Camunda Modeler 4.0 is ready for release.

### CODE REVIEW

File: content/installation/camunda-modeler.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ Reported to work on these operating systems, too:
 
 For executing BPMN Diagrams created using Camunda Modeler, Process Engine version 7.4.0, 7.3.3, 7.2.6 and above is required.
 
-Note that you do not need to install the Process Engine if you do not want to execute the BPMN Models.
+For evaluating DMN Decisions created using Camunda Modeler, Process Engine version 7.13.0 and above is required.
```
Comment (nikku) [inline content/installation/camunda-modeler.md]: Maybe change to include the DMN version "DMN 1.3" produced by the tool and mention all Camunda Platform releases (patches of older releases, too) that support it.

File: content/modeler/dmn.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ The demo above shows how to model a decision table. After creating a decision an
 
 {{< img src="img/save.png" title="Save DMN Diagram" >}}
 
-To save your state of work, click *File / Save File As...* in the top-level menu. Then select a location on your file system to store the diagram in the DMN 1.1 XML format. You can load that file again by clicking *File / Open File...*.
+To save your state of work, click *File / Save File As...* in the top-level menu. Then select a location on your file system to store the diagram in the DMN 1.3 XML format. You can load that file again by clicking *File / Open File...*.
 
-## DMN 1.1 Coverage
+## DMN 1.3 Coverage
```
Comment (nikku) [inline content/modeler/dmn.md]: Maybe drop DMN standard version in this heading alltogether.

File: content/modeler/dmn.md [hunk 3]
```diff
@@ -8,23 +8,23 @@ menu:
...
     parent: "modeler"
-    pre: "How to model executable DMN 1.1 decision diagrams (DRD) and tables."
+    pre: "How to model executable DMN 1.3 decision diagrams (DRD) and tables."
 
 ---
 
 # Quickstart
 
-## Create new DMN 1.1 Decision Requirement Diagram
+## Create new DMN 1.3 Decision Requirement Diagram
```
Comment (nikku) [inline content/modeler/dmn.md]: Maybe drop version from header alltogether.

Comment (barmac) [reply content/modeler/dmn.md]: Makes sense.

File: content/installation/camunda-modeler.md (hunk 1 above)
Comment (barmac) [reply content/installation/camunda-modeler.md]: Sure, makes sense.

Comment (barmac) [reply content/installation/camunda-modeler.md]: Done.

File: content/installation/camunda-modeler.md [hunk 4]
```diff
@@ -37,7 +37,9 @@ Reported to work on these operating systems, too:
 
 For executing BPMN Diagrams created using Camunda Modeler, Process Engine version 7.4.0, 7.3.3, 7.2.6 and above is required.
 
-Note that you do not need to install the Process Engine if you do not want to execute the BPMN Models.
+For evaluating DMN 1.3 Decisions created using Camunda Modeler, Process Engine version 7.13.0, 7.12.4, 7.11.11, 7.10.17 and above is required.
+
+Note that you do not need to install the Process Engine if you do not want to execute the BPMN Models or evaluate DMN Decisions.
```
Comment (philippfromme) [inline content/installation/camunda-modeler.md:42]: I'd rename _BPMN Models_ to _BPMN Diagrams_ as we refer to them as such above.

Comment (barmac) [reply content/installation/camunda-modeler.md]: Done.

**Pull request merged**